compiler/z/codegen/FPTreeEvaluator.cpp

```diff
@@ -1241,8 +1241,9 @@ FPtoIntBitsTypeCoercionHelper(TR::Node * node, TR::CodeGenerator * cg)
           generateLoadLiteralPoolAddress(cg, node, litBase);
           }
 
-       TR::MemoryReference * NaN              = nodeType == TR::Int64 ? generateS390MemoryReference((int64_t)CONSTANT64(0x7ff8000000000000), TR::Int64, cg, litBase) : 
-                                                                           generateS390MemoryReference((int32_t)0x7fc00000, TR::Int32, cg, litBase);
+       TR::MemoryReference * NaN = nodeType == TR::Int64 ?
+                 generateS390MemoryReference((int64_t)CONSTANT64(0x7ff8000000000000), TR::Int64, cg, litBase) :
+                 generateS390MemoryReference((int32_t)0x7fc00000,                     TR::Int32, cg, litBase);
```

Comment:
Please see my previous review suggestion on this line. Let's avoid C-style casts and these awful `CONSTANT64` macros.

Reply:
Ahh, apologies! I thought you were strictly referring to an indentation issue when you mentioned new formatting, so I didn't notice the other suggestions.

Adressed in : 0e2326e20